### ISSUE TITLE
[train-161][stage] 500 error thrown on response validation

### ISSUE THREAD

**Issue by jrgm** [mozilla/fxa#4449]
I was loading the /settings page in stage with v1.161.0 and got a 500 ISE.

Auth-server logs record this stack trace (reformatted):
```
{
  "Timestamp":1583478513469000000,
 "Logger":"fxa-auth-server",
  "Type":"request.summary",
  "Severity":2,
  "Pid":1,
  "EnvVersion":"2.0",
  "Fields":{
    "status":500,
    "errno":999,
    "path":"/v1/account",
    "lang":"en-US,en;q=0.5",
    "agent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10.14; rv:73.0) Gecko/20100101 Firefox/73.0",
    "remoteAddressChain":"[\"67.3.3.3\",\"172.31.3.33\",\"127.0.0.1\"]",
    "t":7,
    "uid":"ece2d075da5d4b5fad5ff5b5adf7aff4",
    "keys":false,
    "method":"get",
    "email":"someone@example.com",
    "stack":"ValidationError: child \"subscriptions\" 
fails because [\"subscriptions\" at position 0 
fails because [child \"plan_name\" 
fails because [\"plan_name\" must be a string]]]
    at Object.exports.process (/app/node_modules/@hapi/joi/lib/errors.js:202:19)
    at internals.Object._validateWithOptions (/app/node_modules/@hapi/joi/lib/types/any/index.js:763:31)
    at internals.Object.validate (/app/node_modules/@hapi/joi/lib/types/any/index.js:797:21)
    at Object.internals.validate (/app/node_modules/@hapi/hapi/lib/validation.js:233:19)
    at exports.response (/app/node_modules/@hapi/hapi/lib/validation.js:204:37)
    at Request._postCycle (/app/node_modules/@hapi/hapi/lib/request.js:455:68)
    at Request._reply (/app/node_modules/@hapi/hapi/lib/request.js:434:20)
    at Request._execute (/app/node_modules/@hapi/hapi/lib/request.js:275:14)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)"}}
```

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXA-1309)
┆Issue Number: FXA-1309

**Comment by madalincm** [comment]
@jrgm We have just encountered this https://github.com/mozilla/fxa/issues/4450 while trying to create a subscription. Are this 2 issues related?

**Comment by madalincm** [comment]
@jrgm We have just encountered this https://github.com/mozilla/fxa/issues/4450 while trying to create a subscription. Are this 2 issues related?

**Comment by jrgm** [comment]
Yes, same bug.

**Comment by bbangert** [comment]
Closed via #4455.

**Comment by madalincm** [comment]
I have retested stage today and the issue is no longer reproducing. Marking issue as verified fixed.
